libbeat/scripts/Makefile

```diff
@@ -35,7 +38,7 @@ TEST_ENVIRONMENT?=false
 SYSTEM_TESTS?=false
 GOX_OS?=linux darwin windows solaris freebsd netbsd openbsd
 DOCKER_COMPOSE?=docker-compose -f docker-compose.yml
-
+GOPACKAGES_COMMA_SEP=$(subst $(space),$(comma),$(strip ${GOPACKAGES}))
```

Comment:
Looks like quite a bit of magic. I didn't test if the line works but as the builds still work, LGTM.